opm/grid/cpgrid/CpGridData.hpp

```diff
@@ -257,6 +285,592 @@ class CpGridData
         ijk[2] = gc / logical_cartesian_size_[1];
     }
 
+private:
+    /// @brief
+    ///   Compute amount of cells in each direction of a patch of cells. (Cartesian grid required).
+    ///
+    /// @param [in]  startIJK  Cartesian triplet index where the patch starts.
+    /// @param [in]  endIJK    Cartesian triplet index where the patch ends.
+    ///
+    /// @parem [out] patch_dim Patch dimension {#cells in x-direction, #cells in y-direction, #cells in z-direction}.
+    const std::array<int,3> getPatchDim(const std::array<int,3>& startIJK, const std::array<int,3>& endIJK) const
+    {
+        return {endIJK[0]-startIJK[0], endIJK[1]-startIJK[1], endIJK[2]-startIJK[2]};
+    }
+
+    /// @brief
+    ///   Compute corner, face, and cell indices of a patch of cells. (Cartesian grid required).
+    ///
+    /// @param [in]  startIJK  Cartesian triplet index where the patch starts.
+    /// @param [in]  endIJK    Cartesian triplet index where the patch ends.
+    ///
+    /// @param [out] {patch_corners, patch_faces, patch_cells} Indices of corners, faces, and cells of the patch of cells.
```

Comment:
Is this correct? I don't see this parameter.
Maybe add comment about what is returned: ` /// @return description`

Reply:
It seems I confused what @param [out] is used for. I guess that when a function returns something, we just document it as @return (and not @param [out]).